public/js/login.js

```diff
@@ -0,0 +1,37 @@
+const email = document.getElementById('email');
+const password = document.getElementById('password');
+const btnLogin = document.getElementById('btn_login');
+const msg = document.getElementById('msg');
+
+btnLogin.addEventListener('click', (e) => {
+  e.preventDefault();
+  const emailV = email.value.trim();
+  const passwordV = password.value;
```

Comment:
why you didn't add trim to password value
```suggestion
  const passwordV = password.value.trim();
  
```